tests/error/comprehension_errors/used_twice2.err

```diff
@@ -1,7 +1,7 @@
 Guppy compilation failed. Error in file $FILE:18
 
 16:    @guppy(module)
-17:    def foo(qs: linst[qubit]) -> linst[qubit]:
+17:    def foo(qs: linst[qubit] @owned) -> linst[qubit]:
 18:        return [q for q in qs if bar(q)]
-                   ^
-GuppyError: Variable `q` with linear type `qubit` was already used (at 18:33)
+                                        ^
+GuppyTypeError: Variable `q` with linear type `qubit` is not used on all control-flow paths of the list comprehension
```

Comment:
You need to annotate `bar` as `owned` as well to reproduce the old error

Reply:
Good spot! Updated